README.md

```diff
@@ -20,6 +20,8 @@ The next step after provisioning an HPCS instance is to [initialize](https://clo
 For more information, see [components and concepts](https://cloud.ibm.com/docs/hs-crypto?topic=hs-crypto-understand-concepts) of HPCS and [about service instance initialization](https://cloud.ibm.com/docs/hs-crypto?topic=hs-crypto-introduce-service) in the Cloud Docs.
 
 
+If you provision HPCS instance with `private-only` endpoint then you receive the `public` and `private` endpoints URL in the output. Only the private endpoint works. The public endpoint is just for reference and only valid if public-and-private endpoint is setup.
```

Comment:
```suggestion
If you provision an HPCS instance with a `private-only` endpoint, both `public` and `private` endpoints URLs are included in the output. You can disregard the public endpoint.
```

I'm assuming there's no real reason for the public endpoint in this situation. If there is, can we be specific about why it's included? What does the public endpoint refer to if it's "just for reference"?

Reply:
Added some more details. Please review if that helps.